### PR TITLE
🍒  [23.10.2] DU ML models

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/UiPathDocumentOCR_CPU__11__metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "changeLog":  "",
+    "cpu":  0,
+    "description":  "Machine Learning model(available in Preview) for extracting text and checkboxes/radio buttons from Documents. Please see more details including supported languages and link to Activities guide in the Document Understanding Guide here: https://docs.uipath.com/document-understanding/v2021.4/docs/ml-packages#uipath-document-ocr. For alternative deployment methods in high-volume scenarios see also this page: https://docs.uipath.com/document-understanding/v2021.4/docs/ocr-services",
```
Comment (bogdan-dumitru-uipath) [inline metadata/UiPathDocumentOCR_CPU__11__metadata.json:4]: nit: missing ` ` (space) before `(available` in `Machine Learning model(available in Preview)`

File: metadata/Passports__16__metadata.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "description":  "Machine Learning model for extracting commonly occurring data points from Passports, including header fields and line items. Please see more details including supported languages in the ML Packages section of the Document Understanding Guide: https://docs.uipath.com/document-understanding/docs/ml-packages",
+    "displayName":  "Passports",
+    "gpu":  0,
+    "inputDescription":  "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -\u003e Other Services view.",
+    "inputType":  "JSON",
+    "memory":  0,
+    "mlPackageLanguage":  "PYTHON37_DU",
+    "name":  "Passports",
+    "outputDescription":  "Please refer to the documentation of the Activity used to query the ML Skill.",
+    "processorType":  "GPU",
```
Comment (bogdan-dumitru-uipath) [inline metadata/Passports__16__metadata.json:13]: this is the only head / doctype that says `"processorType": "GPU"` (all the rest say `"CPU"`), is this ok? 👀

**Review by bogdan-dumitru-uipath — APPROVED** [review]
pls see comments

File: metadata/Passports__16__metadata.json (hunk 2 above)
Comment (irinel-birsan-uipath) [reply metadata/Passports__16__metadata.json:13]: It is informative only. Fixed!

File: metadata/UiPathDocumentOCR_CPU__11__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/UiPathDocumentOCR_CPU__11__metadata.json:4]: Done!

**Review by bogdan-dumitru-uipath — APPROVED** [review]

**Review by sstefanov78 — APPROVED** [review]

**Pull request merged**